apps/framework-cli/deploy/kubernetes/moose-deployment.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: webserver-deployment
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: mooseweb
+  template:
+    metadata:
+      labels:
+        app: mooseweb
+        ver: v1
+    spec:
+      containers:
+      - name: web
+        image: pnxtech/moose-deployment-x86_64-unknown-linux-gnu:latest
```

Comment:
You don't have access to the 514 org?

Reply:
No not yet.  I remember we discussed it - just don't remember the outcome. Will update when I gain access.